src/js/_enqueues/wp/theme.js

```diff
@@ -349,7 +349,7 @@ themes.Collection = Backbone.Collection.extend({
 		request: {}
 	},
 
-	// Send request to api.wordpress.org/themes.
+	// Send request to the `/themes` endpoint of the WordPress API.
```

Comment:
```suggestion
	// Send request to the `/themes` endpoint of the WordPress.org API.
```
In the inline documentation (here and elsewhere), could we keep the WordPress.org API reference (since that's the default)? “WordPress API” seems a bit vague to me, and could be confused with the REST API, for example.

Reply:
Valid point, definitely open to doing so, as the first pass on those was to keep it in line with the idea that it may be passed via another location first, but I'm very flexible on this :+1: